kinder/ci/workflows/kustomize-1.20.yaml

```diff
@@ -0,0 +1,9 @@
+version: 1
+summary: |
+  This workflow implements a sequence of tasks used for testing usage of kustomize patch with kubeadm.
+  test grid > https://testgrid.k8s.io/sig-cluster-lifecycle-kubeadm#kubeadm-kinder-kustomize-1-20
+  config    > https://github.com/kubernetes/test-infra/blob/master/config/jobs/kubernetes/sig-cluster-lifecycle/kubeadm-kinder-kustomize.yaml
+vars:
+  kubernetesVersion: "{{ resolve `ci/latest-1.20` }}"
+tasks:
+- import: kustomize-tasks.yaml
```

Comment:
we can delete this file, because the kustomize feature is being removed in 1.20.

Reply:
sure

Reply:
deleted.